### DISCUSSION TITLE
Can i get the full source code?

### DISCUSSION THREAD

**Discussion by JordanGithub7** [RocketChat/Rocket.Chat#30722] [General]
As an enthusiast for Instant Messaging, I'm interested in accessing the source code. Are there any limitations on the free version, or do I need to go for a paid subscription to get full access? I'm willing to pay, but I need clarity on what source code I'll have access to after payment and how paid versions differs from the free version in terms of source code.

**Comment by Kathrininfanta** [comment]
Hey,
If you’re interested in building an instant messaging solution, it can take a long time to develop from scratch. Some providers offer instant messaging APIs and SDKs that you can integrate with your software. These solutions are customizable based on your requirements and are secure.

Check this blog: [Best Instant Messaging for Business](https://www.apptha.com/blog/instant-messaging-for-business/)
